tendrl/monitoring_integration/graphite/graphite_utils.py

```diff
@@ -1,9 +1,47 @@
+import ConfigParser
 import copy
-
+import os
 
 from tendrl.commons.utils import log_utils as logger
 
 
+def archive(
+    resource_path,
+    archive_path,
+    resource_name,
+    resource_type
+):
+    ret_val = os.system(
+        "mv " + str(resource_path) + " " + str(archive_path)
+    )
+    if ret_val is 0:
+        msg = "%s - %s deleted from graphite" % (
+            resource_type, resource_name
+        )
+        logger.log("info", NS.get("publisher_id", None),
+                   {'message': msg})
+    else:
+        msg = "%s - %s deleted from graphite failed" % (
+            resource_type, resource_name
+        )
+        logger.log("error", NS.get("publisher_id", None),
+                   {'message': msg})
+
+
+def get_data_dir_path():
+    carbon_path = "/etc/carbon/carbon.conf"
```

Comment:
I remember this is configurable

Reply:
i will change it to configure from monitoring_integration carbon.conf